### ISSUE TITLE
Electric motors stop working after server restart

### ISSUE THREAD

**Issue by quickrick2** [mrh0/createaddition#71]
**Describe the bug**
I am running a server with a few mods and am using electric motors to power a lot of my machines, whenever I restart the server some of them stop turning and I have to break and place them again.

**To Reproduce**
Restart the server with a large number of different motors running.

**Additional context**
- Mod version: createaddition-1.16.5-20210517a
- Forge version: forge-36.1.23
- Create version: create-mc1.16.5_v0.3.1c
- Complete modlist: 
![image](https://user-images.githubusercontent.com/54983100/120294219-4876f200-c2c6-11eb-979d-b68fc77477ee.png)

**Comment by dannydjdk** [comment]
I have noticed this same phenomenon when the motor temporarily runs out of power. If I turn the speed way down, the motor will re-engage, and then I can turn it back up again. This may be the same bug. What might be happening is that when the server reboots, there may be a brief interruption in rf. I wonder if your motors would restart if you just turned the speed down and then back up rather than breaking and replacing.

**Comment by mrh0** [comment]
I'll look into it

**Comment by dannydjdk** [comment]
It seems there was another factor affecting the case where my motors would not restart after losing power. I had increased the power consumption of the motor without increasing the rf capacity or max input. So, it was barely able to accept power fast enough to keep the motors turning.

Interestingly though, it WAS able to accept power fast enough as long as the power kept coming, but once it lost power, it couldn't recover until I turned the speed way down and then back up again. Increasing the motor capacity and max input seems to have solved the issue, as I can no longer reproduce it.

Whether or not this is related to the OP's issue is unclear. If the OP can reproduce the issue and fix it by turning the speed down and then back up, then it likely is related.
